docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -179,6 +179,10 @@ Person.delete("status", Status.Alive);
 
 // delete all persons
 Person.deleteAll();
+
+// update all living persons
+Person.update("set name = 'Moral' where status = ?1", Status.Alive);
```

Comment:
You should add documentation about the new update queries in the `Simplified queries` section as well as in `package-info.java` javadoc.

Reply:
It's still missing in the `Simplified queries` section of this guide.

Reply:
Done.